README.md

```diff
@@ -156,7 +156,7 @@ models:
           expression: "col_a + col_b = total"
 ```
 
-The macro accepts an optional argument `condition` that allows for asserting
+The macro accepts an optional argument `where` that allows for asserting
```

Comment:
```suggestion
```
This is true for all tests, so I think calling it out here is probably more confusing than is necessary.